mappings/net/minecraft/Formatting.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS c net/minecraft/ChatFormat
+CLASS c net/minecraft/Formatting
```

Comment:
Should this go in the `text` package as well?

Reply:
It is used by more than text, as implicit toString within concatenation treats it as a literal Unicode formatting. Maybe it should be in the `util` package though, I overlooked that.